src/IIIFPresentation/API/Converters/CollectionConverter.cs

```diff
@@ -1,20 +1,18 @@
-﻿using API.Infrastructure.Helpers;
+﻿using Core.Helpers;
 using IIIF.Presentation.V3;
-using IIIF.Presentation.V3.Annotation;
-using IIIF.Presentation.V3.Content;
 using Models.API.Collection;
 using Models.Infrastucture;
 
 namespace API.Converters;
 
 public static class CollectionConverter
 {
-    public static Collection ToHierarchicalCollection(this Models.Database.Collections.Collection dbAsset, UrlRoots urlRoots,
-        IQueryable<Models.Database.Collections.Collection>? items)
+    public static Collection ToHierarchicalCollection(this Models.Database.Collections.Collection dbAsset, 
```

Comment:
There's a lot of Id/Url wrangling in this class - could those be in a central place for reuse? Would make any changes easier to apply too